### PR TITLE
feat: use neutronjs in sdk47 #ntrn-147

### PR DESCRIPTION
**Pull request by NeverHappened**
https://hadronlabs.atlassian.net/browse/NTRN-147

What's done:
1. Uses buf to generate protobuf generated files
2. Moved all newer sdk47 branch code for sdk47 into neutronjs
3. Removed all helper's code from integration tests

Integration tests: https://github.com/neutron-org/neutron-integration-tests/pull/244
NeutronJS: https://github.com/neutron-org/neutronjsplus/pull/11

### CODE REVIEW

**Comment by NeverHappened** [comment]
Testrun: https://github.com/neutron-org/neutron-tests/actions/runs/7020210258

**Comment by NeverHappened** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/7046159446

**Review by pr0n00gler — APPROVED** [review]
Look good to me

**Review by swelf19 — APPROVED** [review]

**Review by quasisamurai — APPROVED** [review]
👍

**Comment by NeverHappened** [comment]
added parameters to the test:run_in_band
https://github.com/neutron-org/neutron-tests/actions/runs/7113325728/job/19365568183

File: src/testcases/parallel/simple.test.ts [hunk 1]
```diff
@@ -446,7 +447,7 @@ describe('Neutron / Simple', () => {
       });
     });
 
-    describe('Failing sudo handlers', () => {
+    describe('Not enough amount of tokens on contract to pay fee', () => {
```
Comment (sotnikov-s) [inline src/testcases/parallel/simple.test.ts:450]: looks like a change by a mistake that shouldn't have been made

```suggestion
    describe('Failing sudo handlers, () => {
```

**Review by sotnikov-s — APPROVED** [review]

**Pull request merged**